packages/otel/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import { SpanKind, SpanStatusCode, type TracerProvider, trace } from '@opentelemetry/api'
+import {
+  ATTR_HTTP_REQUEST_HEADER,
+  ATTR_HTTP_REQUEST_METHOD,
+  ATTR_HTTP_RESPONSE_HEADER,
+  ATTR_HTTP_RESPONSE_STATUS_CODE,
+  ATTR_URL_FULL,
+  ATTR_HTTP_ROUTE,
+} from '@opentelemetry/semantic-conventions'
+import { createMiddleware } from 'hono/factory'
+import type { Env, Input } from 'hono'
+
+const PACKAGE_NAME = '@hono/otel'
+const PACKAGE_VERSION = '0.1.0'
```

Comment:
I wanted to eliminate the duplication with the metadata in package.json, but couldn't think of a good way to do it.